backend/ohq/vector_db.py

```diff
@@ -0,0 +1,146 @@
+import json
+import pinecone
+import openai
+import time
+import tiktoken
+import unidecode
+
+from langchain.text_splitter import RecursiveCharacterTextSplitter
+
+from ohq.models import Document
+
+openai.api_key = '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'
+encoding = tiktoken.get_encoding("cl100k_base")
+
+pinecone.init(api_key='2a81b230-ba5d-4290-a3ac-b60e6ad17a66', environment='gcp-starter')
```

Comment:
You should put all the API keys in the `.env` file and get the secrets back via `os.getenv`. Strongly recommend against committing keys without using `.env` and without adding `.env` into `.gitignore` when we are committing into a public repo!

Reply:
I think a cleaner way to do this would be to also set up the keys inside the `django.conf `object. You can look at the sms.py to see how it was setup but essential, the steps are:

1. Go to `officehoursqueue/settings/base.py` and define your environment secret keys there (follow the TWILIO example and set up the keys there)and it would look like this `PINECONE_API_KEY=os.environ.get("PINECONE_API_KEY")`
2. Now you can just import the global settings object like this: `from django.conf import settings` or look at `ohq/sms.py`.
3. Get the secret keys by calling` settings.PINECONE_API_KEY` and load it as needed